dispersy.py

```diff
@@ -693,6 +715,8 @@ def get_communities(self):
         """
         return self._communities.values()
 
+    @inlineCallbacks
+    # TODO(Laurens): This method is never called?
```

Comment:
No, then you would find at least a reference (not per se a call) to this function where it is stored in a map or something. Not even that.